akka-docs/src/main/paradox/typed/testing.md

```diff
@@ -202,6 +202,18 @@ Java
 Note that you can add `import testKit._` to get access to the `spawn` and `createTestProbe` methods at the top level
 without prefixing them with `testKit`.
 
+### Observing mocked behavior
+
+If you want to mock and observe some behavior, as it was possible with the untyped `TestActor`/`AutoPilot`, you can use
+the following pattern:
```

Comment:
Isn't this only about intercepting and observing behavior, rather than mocking anything?

Reply:
No, it's about simulating / "mocking" an actor _and_ observing the interaction with him, see #26116

In the text I assumed the reader was coming from untyped `TestActor` / `AutoPilot` looking for a replacement. 
Maybe the description / motivation has to be more elaborate?

Reply:
If it is also about mocking, and we define mocking as creating a behavior that accepts some message interactions without doing any actual logic, this definitely needs to be both more explicit in the text and included in the samples. I don't think an empty accept anything behavior is good enough for this, it should probably show a request-response interaction for completeness.

It would be good if this section started describing the use case and then possibly have the "This is will allow the same kinds of tests as untyped TestActor/Autopilot"

How starting out with something along the lines of:
"When testing an actor that interacts with other actors it can be useful to not have to run the other actors it depends on but to create mock behaviors that accept and possibly respond to messages in the same way the other actor would do but without executing any actual logic. In addition to this it can also be useful to observe those interactions to assert that the actor under test did send the expected messages." 

It could even make sense to do it in a two step fashion where the mocking is first described and then an addon on observing the message interchanges using `monitor`

Reply:
Thanks for your feedback and suggestion, makes sense!

I've now used a more elaborate example and improved the description based on your suggestion.